src/routes/rgbpp/address.ts

```diff
@@ -200,6 +218,49 @@ const addressRoutes: FastifyPluginCallback<Record<never, never>, Server, ZodType
         xudtBalances[type_hash].pending_amount = BI.from(xudtBalances[type_hash].pending_amount)
           .add(BI.from(amount))
           .toHexString();
+      });
+
+      // Find spent RgbppLock Xudt assets in unconfirmed transactions' inputs
+      const allTxs = await fastify.bitcoin.getAddressTxs({ address: btc_address });
```

Comment:
`GET /api/address/:address/txs` only returns up to 50 mempool transactions plus the first 25 confirmed transactions.

ref: https://mempool.space/docs/api/rest#get-address-transactions

Reply:
Okay, this is unexpected. Should I just fetch them page by page using this API until the response contains zero unconfirmed transactions? The other API doesn't seem fit for this feature, because it only returns the first 50 unconfirmed transactions without providing a pagination parameter: https://mempool.space/docs/api/rest#get-address-transactions-mempool

Reply:
An address with more than 50 pending BTC transactions is uncommon.
I think it is a rare case.

What about just explain this situation in the API description -> https://github.com/ckb-cell/btc-assets-api/blob/95fac7e5f9f7ddf2ed3d0c6b74693fda27732654/src/routes/rgbpp/address.ts#L128

e.g.
An address with more than 50 pending BTC transactions is uncommon. However, if such a situation arises, it potentially affecting the returned `total_amount`.

Reply:
Updated description at 2119320f